dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java

```diff
@@ -17,7 +17,7 @@
 package com.google.cloud.tools.opensource.classpath;
 
 /** The kind of linkage error against a symbol reference. */
-enum Reason {
+enum ErrorType {
```

Comment:
In future PRs, try to do these sorts of renames first, rather than mixing it with other changes.

Reply:
Sure, will do.